.ci/pytorch/test.sh

```diff
@@ -1551,6 +1551,7 @@ test_linux_aarch64() {
   python test/run_test.py --include test_modules test_mkldnn test_mkldnn_fusion test_openmp test_torch test_dynamic_shapes \
         test_transformers test_multiprocessing test_numpy_interop test_autograd test_binary_ufuncs test_complex test_spectral_ops \
         test_foreach test_reductions test_unary_ufuncs test_tensor_creation_ops test_ops test_cpp_extensions_open_device_registration \
+        distributed/elastic/timer/api_test distributed/elastic/timer/local_timer_example distributed/elastic/timer/local_timer_test \
```

Comment:
how come the tests being added if they are going to be filtered out anyways?

Reply:
We want to enable the full test suite eventually, so we want to ensure they're all being skipped, xfailed or passing properly in CI ahead of that.

At some point it would be good to revisit this once we use a non-conda Python to try.